<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-7UA5NPA" name="Test" tests="1" failures="0" timestamp="28 Dec 2019 04:13:18 GMT" time="0.966" errors="0">
  <testcase name="test_case" time="0.951" classname="com.Auto_Test.Case.Test_Case_AutoInterface">
    <failure type="java.lang.Throwable">
      <![CDATA[java.lang.Throwable
	at message : expected [Post] but found [Set] in method : .test_case(Test_Case_AutoInterface:59)
	at com.Auto_Test.Case.Test_Case_AutoInterface.test_case(Test_Case_AutoInterface.java:59)
]]>
    </failure>
  </testcase> <!-- test_case -->
</testsuite> <!-- Test -->
